airflow/settings.py

```diff
@@ -80,7 +80,7 @@
 DAGS_FOLDER: str = os.path.expanduser(conf.get('core', 'DAGS_FOLDER'))
 
 engine: Optional[Engine] = None
-Session: Callable[..., SASession]
+Session: SASession
```

Comment:
Let’s use the `Callable` form (it’s done after a discussion in another PR)

Reply:
mypy is complaining for callable:
```
airflow/www/views.py:546: error: "Callable[..., Any]" has no
attribute "bind"
    ... extra_args['sqlite_warning'] = settings.Session.bind...
                                       ^
Found 1 error in 1 file (checked 1 source file)
```

Reply:
I guess w need my new fancy toy then!

```python
class SessionMaker(Protocol):
    """Fake the type returned by sessionmaker for typg hints.

    Add more things to this class as we need them.
    """

    bind: Engine

    def __call__(self, **kwargs: Any) -> SASession:
        ...

    def remove(self) -> None:
        ...
```

Reply:
Or change usages of `settings.Session.bind` to `settings.engine`

Reply:
The downside of `settings.engine` is it’s an `Optional`. Pick your poision I guess :p

Reply:
Nice. Updated with settings.engine